internal/storage/postgres/watering_plan/watering_plan.go

```diff
@@ -104,6 +104,13 @@ func WithTrailer(trailer *entities.Vehicle) entities.EntityFunc[entities.Waterin
 	}
 }
 
+func WithCanellationNote(cancellationNote string) entities.EntityFunc[entities.WateringPlan] {
+	return func(wp *entities.WateringPlan) {
+		slog.Debug("updating cancellation note", "canellation note", cancellationNote)
```

Comment:
Typo: `cancellation`

```suggestion
		slog.Debug("updating cancellation note", "cancellation note", cancellationNote)
```